### PR TITLE
Feature/css 

### PR DESCRIPTION
**Pull request by WalterRestifo**
add a new color palette and improve the nav bar with flexbox

### CODE REVIEW

File: css/styles.css [hunk 1]
```diff
@@ -1,14 +1,115 @@
...
+  font-family: "caveat-bold";
+  src: url("../fonts/Caveat/static/Caveat-Bold.ttf");
+}
+* {
+  box-sizing: border-box;
+}
+
 :root {
   margin: 0px;
   padding: 0px;
```
Comment (MerleMak) [inline css/styles.css:15]: Hi Walter :)
Its unusual to add `padding: 0px;` and `margin : 0px;` to the root element and also unnecessary as it doesn't have any default style. The :root is usually used to define variables for color values for example. If you want to add `padding: 0px;` and `margin : 0px;` you can add them to the universal Selector `* { }`
```suggestion

```

File: css/styles.css [hunk 2]
```diff
@@ -1,14 +1,115 @@
...
+  background-color: rgb(249, 249, 240);
+  border-radius: 20px;
+}
+
+.nav-parent {
+  display: flex;
+  list-style: none;
+  justify-content: space-around;
+  flex-wrap: wrap;
+
```
Comment (MerleMak) [inline css/styles.css:50]:
```suggestion
```

File: index.html [hunk 3]
```diff
@@ -11,17 +11,19 @@
...
+        </ul>
       </nav>
     </header>
     <main>
       <figure>
         <img
           class="profile-img"
-          src="https://ca.slack-edge.com/TTHG21AH3-U045KCN06H1-5d38fda5992d-72"
+          src="assets/IMG_20210613_220316_772.jpg"
           alt="Image of Walter Restifo with his best smile"
```
Comment (MerleMak) [inline index.html:27]: 😁👍🏼

File: index.html [hunk 4]
```diff
@@ -71,7 +73,13 @@ <h2>Curriculum Vitae</h2>
...
         <p>E-mail: walterrestifo93@gmail.com</p>
         <p>Mobile: 0171 7258168</p>
-        <p>Instagram: Walter Restifo</p>
+        <p>
+          <a
+            class="instagram"
+            href="https://www.instagram.com/walterrestifo93/?hl=de"
+            >Instagram: Walter Restifo</a
+          >
+        </p>
```
Comment (MerleMak) [inline index.html:82]:
```suggestion
          <a
            class="instagram"
            href="https://www.instagram.com/walterrestifo93/?hl=de"
            >
            Instagram: Walter Restifo</a>
```

**Review by MerleMak — CHANGES_REQUESTED** [review]
Hi Walter, great job!
I left some remarks you might want to consider!
Keep it up! 🙌🏽

File: index.html (hunk 4 above)
Comment (WalterRestifo) [reply index.html]: I´m not sure if i really understand what you mean... please take a look at the changes I made

File: css/styles.css (hunk 1 above)
Comment (WalterRestifo) [reply css/styles.css]: thank you, I made it and I write a few color vars in root

**Pull request merged**